superset/utils/decorators.py

```diff
@@ -51,6 +51,7 @@ def etag_cache(
     check_perms: Callable[..., Any],
     get_last_modified: Optional[Callable[..., Any]] = None,
     skip: Optional[Callable[..., Any]] = None,
+    must_revalidate: Optional[bool] = False,
```

Comment:
We could potentially always assume `must_revalidate` if `get_last_modified` is provided. Because validation only makes sense when there is a way to programmatically get a resource's the "real" last modified time.